terraform/hub/manifets/clusterStore.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: aws-secrets-manager
+  namespace: external-secrets
+spec:
+  provider:
+    aws:
+      service: SecretsManager
+      region: eu-west-1
```

Comment:
This should be a parameter.

‘Can we avoid using plain manifest and using a helm chart ?

Reply:
Yes those will be updated on the second revision we will make incremental changes first platform then gitops appsets